modules/RtcStats/RtcStats.js

```diff
@@ -0,0 +1,138 @@
+import Logger from '@jitsi/logger';
+import rtcstatsInit from '@jitsi/rtcstats/rtcstats';
+import traceInit from '@jitsi/rtcstats/trace-ws';
+
+const logger = Logger.getLogger(__filename);
+
+/**
+ * Filter out RTCPeerConnection that are created by callstats.io.
+ *
+ * @param {*} config - Config object sent to the PC c'tor.
+ * @returns {boolean}
+ */
+function connectionFilter(config) {
+    if (config?.iceServers[0] && config.iceServers[0].urls) {
+        for (const iceUrl of config.iceServers[0].urls) {
+            if (iceUrl.indexOf('callstats.io') >= 0) {
+                return true;
+            }
+        }
+    }
+}
+
+/**
+ * Class the controls the rtcstats node module.
+ */
+class RtcStats {
+    initialized = false;
+    trace = null;
+    eventCallback = null;
+    closeCallback = null;
+    setTrace;
+    traceConfigured = new Promise(r => {
+        this.setTrace = r;
+    });
+    meetingFqn = null;
+    setMeetingF;
+    meetingFqnSet = new Promise(r => {
+        this.setMeetingF = r;
+    });
+
+    /**
+     * Initializes the underlying rtcstats node moduke.
+     */
+    init(options) {
```

Comment:
There needs to be a way to "de-init" since mobile needs that.

Reply:
Is there a specific place this de-init needs to be called from?

Reply:
Init will be called a second time, so as long as that works we are good!

Reply:
Oh yes even on the web this gets initialized several times. I haven't looked into the why, but it works.